README.md

```diff
@@ -1,5 +1,5 @@
 [![Build
-Status](https://travis-ci.org/adborden/ballot-api.svg?branch=master)](https://travis-ci.org/adborden/ballot-api)
+Status](https://travis-ci.org/caciviclab/ballot-api.svg?branch=master)](https://travis-ci.org/caciviclab/ballot-api)
```

Comment:
Will we be using the `master` branch for CI? or do we want to use an `integration` (or other name) for it?